styles.css

```diff
@@ -260,6 +266,106 @@ body {
   transition: all 0.8s cubic-bezier(0.075, 0.82, 0.165, 1);
 }
 
+/* project details popup start */
+#project-details-modal {
+  position: fixed;
+  width: 100%;
+  background-color: rgba(3, 3, 3, 0.596);
+  backdrop-filter: blur(6px);
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: 20;
+  padding: 60px 16px 40px 16px;
+  display: none;
+}
+
+#popup-details-card {
+  height: calc(100% + 2px);
+  padding: 16px;
+  background-color: #fff;
+  overflow-y: scroll;
+  border: 1px solid #dfe1e6;
+}
```

Comment:
- [ ] Your design for the popup is fantastic! :+1: However, it is important that the popup is responsive, meaning it should adjust properly to both mobile and desktop screen sizes without the need for a scrollbar. 
- [ ] Furthermore, please ensure that the popup is centered in the middle of the page, and the buttons below take all the with from the popup to align with the template. 

**Your design**

![Screenshot from 2023-03-09 05-23-17](https://user-images.githubusercontent.com/105117832/223998557-b81d2f40-ce12-4253-8ffe-b97fdeca387c.png)

![Screenshot from 2023-03-09 05-38-10](https://user-images.githubusercontent.com/105117832/223999269-a4dd79a7-3563-4e90-9ee5-78699ddd68cd.png)


**Figma**

![Screenshot from 2023-03-09 05-25-33](https://user-images.githubusercontent.com/105117832/223998813-983f4f55-a2b4-4749-8ff3-c5c5aa57a86d.png)